ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.js

```diff
@@ -80,40 +85,33 @@ const ConfirmAddSuggestedToken = () => {
   const tokens = useSelector(getTokens);
   const trackEvent = useContext(MetaMetricsContext);
 
-  const knownTokenActionableMessage = useMemo(() => {
+  const knownTokenBannerAlert = useMemo(() => {
     return (
       hasDuplicateAddress(suggestedTokens, tokens) && (
-        <ActionableMessage
-          message={t('knownTokenWarning', [
+        <BannerAlert severity={Severity.Warning}>
```

Comment:
Let's add a margin top here
```suggestion
        <BannerAlert severity={Severity.Warning}  marginTop={4}>
```